coalib/bearlib/languages/definitions/ABAP.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class ABAP:
+    extensions = '.abap', '.ABAP'
```

Comment:
good question, if pygments people thing they should maybe we should just go with it